### ISSUE TITLE
Scale problem in ADMM / CG

### ISSUE THREAD

**Issue by tbalke** [lanl/scico#38]
@bwohlberg @lukepfister @Michael-T-McCann 

Scaling the f-loss seems to scale the reciprocal of the reconstruction (see below). Initially, I brought this up when designing the `svmbir` PPP example but now I found out that it is not limited to that. It seems like the problem should be somewhere in the CG but not sure.

Lastly, using `subproblem_solver=None` produces `nan` as a result which may or may not be from the same origin.

```python
import jax
import numpy as np
from scico import functional, linop, loss
from scico.admm import ADMM, LinearSubproblemSolver

np.random.seed(123)
N = 10
M = 100

x = jax.device_put(np.random.randn(N))
A = linop.MatrixOperator(np.random.randn(M,N))
y = A @ x

f1 = loss.SquaredL2Loss(y=y, A=A, scale=1)
f42 = loss.SquaredL2Loss(y=y, A=A, scale=42)

x1 = ADMM(
    f=f1,
    g_list=[functional.ZeroFunctional()],
    C_list=[linop.Identity(x.shape)],
    rho_list=[1],
    subproblem_solver=LinearSubproblemSolver(cg_function='scico'), #same with 'jax'
).solve()

x42 = ADMM(
    f=f42,
    g_list=[functional.ZeroFunctional()],
    C_list=[linop.Identity(x.shape)],
    rho_list=[1],
    subproblem_solver=LinearSubproblemSolver(cg_function='scico'), #same with 'jax'
).solve()

print(np.mean(x1/x42))

# 41.99956
```

**Comment by tbalke** [comment]
@lukepfister Yes, that did it!! Will create a PR.

**Comment by bwohlberg** [comment]
Resolved by #45
